src/odemis/gui/win/acquisition.py

```diff
@@ -1233,14 +1244,19 @@ def on_acquire(self, evt):
         if self._main_data_model.opm:
             self._main_data_model.opm.setAcqQuality(path.ACQ_QUALITY_BEST)
 
-        zlevels = self._get_zstack_levels()
+        zlevels, focus_ref = self._get_zstack_levels()
         focus_mtd = FocusingMethod.MAX_INTENSITY_PROJECTION if zlevels else FocusingMethod.NONE
 
         if self.filename_tiles:
             logging.info("Acquisition tiles logged at %s", self.filename_tiles)
             os.makedirs(os.path.dirname(self.filename_tiles), exist_ok=True)
 
         if self.autofocus_roi_ckbox.value:
+            areas = self._get_areas()
+            # calculate relative range of z levels
```

Comment:
Yes. That's quite clear, from the code. What you should explain in this comment is **why** do you want the relative position (while you just had computed the absolute position).